lib/dal/src/workspace_snapshot/graph.rs

```diff
@@ -1030,7 +1029,11 @@ impl WorkspaceSnapshotGraph {
             },
         );
         let filename_no_extension = format!("{}-{}", Ulid::new().to_string(), suffix);
-        let mut file = File::create(format!("/home/zacharyhamm/{filename_no_extension}.txt"))
+
+        let home_str = std::env::var("HOME").unwrap();
+        let home = std::path::Path::new(&home_str);
+
+        let mut file = File::create(home.join(format!("{filename_no_extension}.txt")))
```

Comment:
I don't remember the purpose of the "no extension" thing... I vaguely remember it was due to copy/pasting terminal output, but I think it's fine if you collapsed these together. This is a nit though. Feel free to ignore.